packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -1420,7 +1420,7 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
         return this.labelProvider.getLongName(node);
     }
     protected getDepthPadding(depth: number): number {
-        return depth * this.props.leftPadding;
+        return (depth + 1) * this.props.leftPadding;
```

Comment:
@FernandoAscencio do you mind explaining this change a bit further, I did not see anything in the pull-request description or commit message. Why do we want to increase the depth padding globally for all trees?

Reply:
It has been a while. After a review and some testing I found no difference in display. The `+1` will return to the `file-tree-widget` override.